OneCommunityGlobal/HighestGoodNetworkApp
# Jingyi edit header message

# Description
Gives the user permission to edit the message displayed in the header.

## Related PRS (if any):
This frontend PR is related to the [#985](https://github.com/OneCommunityGlobal/HGNRest/pull/985) backend PR.

## Main changes explained:
- Create a new permission label "editHeaderMessage"
- Check the permission before rendering component in the frontend

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as owner user
5. go to other links -> permission management -> manage user permission
6. Choose a volunteer account and grant this account "edit header message" permission and then submit
7. Log in to the volunteer account who granted this permission
8. Check if this account can edit the header message

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/112589476/4d1cb633-f847-4d31-ad9f-6039cdb6ef34

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/112589476/38de6252-3aa5-45ee-a991-c8463313e3c9
